levels/lovelace_tower/conversations/poster_lovelace.pug

```diff
@@ -0,0 +1,20 @@
+-
+  const STATE_KEY = 'TQ_API_ACADEMY_WORLD_STATE';
```

Comment:
I'm just now realizing that we've been pulling in state differently than we do in event files (this is just an observation and by no means a request for change). If one of the state keys were to change for whatever reason, we would likely run into trouble. @rmkubik are you familiar with this at all?

Event file example
```js
const { STATE_KEY_PROPERTY_NAME } = require('path/to/config');
const worldState = world.getState(STATE_KEY_PROPERTY_NAME);
```

Conversation example
```js
const STATE_KEY = 'SOME_STATE';
const worldState = getState(STATE_KEY);
```